### ISSUE TITLE
Add save/restore buttons on graph view leaf

### ISSUE THREAD

**Issue by Sanqui** [Sanqui/obsidian-persistent-graph#8]
Instead of requiring the use of commands, add save/restore buttons alongside "Close" and "More options".

Method suggested by koala:
>I think you'd be better off using addAction on the graph view leaf https://github.com/obsidianmd/obsidian-api/blob/d4b79f95ccf3838a305915a137a844250766d023/obsidian.d.ts#L1514 and storing a reference to each added button on your plugin. Then you can just call HTMLElement.detach() to remove it.
